files/virt-v2v-wrapper.py

```diff
@@ -229,6 +229,10 @@ def wrapper(data, state_file, v2v_log):
             '-os', data['export_domain'],
             ])
 
+    if 'network_mappings' in data:
+        for mapping in data['network_mappings']:
```

Comment:
please include also check for the type:  `isinstance(data['network_mappings'], list)`

log an error if the type does not match